frontend/src/reunion/TemasHandler.js

```diff
@@ -28,14 +28,20 @@ class TemasHandler extends React.Component {
   }
 
   render() {
+    const tabInicial = () => {
+      if (!this.props.location.state) {
+        return 'Resumen';
+      }
+      return this.props.location.state.tab;
+    };
     return <VistaTemas
       usuario={this.props.usuario}
       temas={this.props.temas}
       dispatch={this.props.dispatch}
       cerrarReunion={this.cerrarReunion}
       estadoReunion={this.props.estadoReunion}
       volverALobby={this.volverALobby}
-      tabInicial={this.props.location.state.tab}
+      tabInicial={tabInicial()}
```

Comment:
Hace falta definir una función? No puede ser directamente...

```suggestion
      tabInicial={this.props.location.state.tab || 'Resumen'}
```